anitya/lib/model.py

```diff
@@ -20,17 +20,28 @@
 anitya mapping of python classes to Database Tables.
 """
 
+try:
+    # The Python 3.6+ API
+    from secrets import choice as random_choice
+except ImportError:
+    from random import choice as random_choice
 import collections
 import datetime
 import logging
 import time
+import string
 
+import six
 import sqlalchemy as sa
-from sqlalchemy.orm import validates
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import (
+    sessionmaker, scoped_session, query as sa_query, validates)
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm.exc import NoResultFound
-from sqlalchemy.orm import sessionmaker, scoped_session, query as sa_query
+from sqlalchemy.types import TypeDecorator, CHAR
+import uuid
```

Comment:
```uuid``` is part of the standard library, so this should be moved to the first import group for PEP-8.